features/search/src/main/java/com/chesire/nekome/app/search/host/core/model/SearchGroup.kt

```diff
@@ -0,0 +1,6 @@
+package com.chesire.nekome.app.search.host.core.model
+
+enum class SearchGroup {
```

Comment:
I think a better name than `SearchGroup` would be good... maybe we could reuse the `SeriesType`, or just create a `SeriesType` inside this module?